Content.Server/Silicons/Laws/IonStormSystem.cs

```diff
@@ -0,0 +1,280 @@
+        var ev = new IonStormLawsEvent(laws);
+        RaiseLocalEvent(ent, ref ev);
+    }
+
+    // for your own sake direct your eyes elsewhere
+    private string GenerateLaw()
+    {
+        // pick all values ahead of time to make the logic cleaner
+        var threats = Pick(Threats);
+        var objects = Pick(Objects);
+        var crew1 = Pick(Crew);
+        var crew2 = Pick(Crew);
+        var adjective = Pick(Adjectives);
+        var verb = Pick(Verbs);
+        var number = Pick(NumberBase) + " " + Pick(NumberMod);
+        var area = Pick(Areas);
+        var feeling = Pick(Feelings);
+        var feelingPlural = Pick(FeelingsPlural);
+        var must = Pick(Musts);
+        var require = Pick(Requires);
+        var action = Pick(Actions);
+        var allergy = Pick(Allergies);
+        var allergySeverity = Pick(AllergySeverities);
+        var concept = Pick(Concepts);
+        var drink = Pick(Drinks);
+        var food = Pick(Foods);
+
+        var joined = $"{number} {adjective}";
+        // a lot of things have subjects of a threat/crew/object
+        var triple = _robustRandom.Next(0, 3) switch
+        {
+            0 => threats,
+            1 => crew1,
+            2 => objects,
+            _ => throw new IndexOutOfRangeException(),
+        };
+        var crewAll = _robustRandom.Prob(0.5f) ? crew2 : Loc.GetString("ion-storm-crew");
+        var objectsThreats = _robustRandom.Prob(0.5f) ? objects : threats;
+        var objectsConcept = _robustRandom.Prob(0.5f) ? objects : concept;
+        // s goes ahead of require, is/are
+        // i dont think theres a way to do this in fluent
+        var (who, plural) = _robustRandom.Next(0, 5) switch
+        {
+            0 => (Loc.GetString("ion-storm-you"), false),
+            1 => (Loc.GetString("ion-storm-the-station"), true),
+            2 => (Loc.GetString("ion-storm-the-crew"), true),
+            3 => (Loc.GetString("ion-storm-the-job", ("job", crew2)), false),
+            _ => (area, true) // THE SINGULARITY REQUIRES THE HAPPY CLOWNS
+        };
+        var jobChange = _robustRandom.Next(0, 3) switch
+        {
+            0 => crew1,
+            1 => Loc.GetString("ion-storm-clowns"),
+            _ => Loc.GetString("ion-storm-heads")
+        };
+        var part = Loc.GetString("ion-storm-part", ("part", _robustRandom.Prob(0.5f)));
+        var harm = _robustRandom.Next(0, 6) switch
+        {
+            0 => concept,
+            1 => $"{adjective} {threats}",
+            2 => $"{adjective} {objects}",
+            3 => Loc.GetString("ion-storm-adjective-things", ("adjective", adjective)),
+            4 => crew1,
+            _ => Loc.GetString("ion-storm-x-and-y", ("x", crew1), ("y", crew2))
+        };
+
+        if (plural) feeling = feelingPlural;
+
+        var subjects = _robustRandom.Prob(0.5f) ? objectsThreats : Loc.GetString("ion-storm-people");
+
+        // message logic!!!
+        return _robustRandom.Next(0, 36) switch
```

Comment:
```suggestion
        return _robustRandom.Next(0, 35) switch
```
35 is excluded, so we get a number from 0-34 which fits the numbers we have in the switch if we correct them (_ corresponds to 34)